crates/node/src/chain_spec.rs

```diff
@@ -89,6 +89,17 @@ pub fn development_config(enable_manual_seal: Option<bool>) -> Result<DevChainSp
     ))
 }
 
+use sp_core::hexdisplay::HexDisplay;
+pub fn print_development_account() {
+    let argent_account_address = Felt252Wrapper::from_hex_be(ARGENT_ACCOUNT_ADDRESS).unwrap();
+    let signer = get_storage_key(&argent_account_address, "_signer", &[], 0);
```

Comment:
the signer here is the public key, we can just take the address and private from the constants file mentioned above